include/seqan3/alphabet/quality/phred42.hpp

```diff
@@ -67,217 +66,40 @@ namespace seqan3
  *
  * \snippet test/snippet/alphabet/quality/phred42.cpp general
  */
-struct phred42
+class phred42 : public quality_base<phred42, 42>
 {
```

Comment:
They are classes, because they are not aggregate types anymore and some have extra constructors. This fixes some long-standing design problems and gives us the opportunity to add custom constructors. The memory behaviour is still the same, because they are still POD, as you said.